api/object.go

```diff
@@ -333,24 +314,21 @@ func (opts HeadObjectOptions) ApplyHeaders(h http.Header) {
 }
 
 func (opts GetObjectOptions) Apply(values url.Values) {
-	if opts.Prefix != "" {
-		values.Set("prefix", opts.Prefix)
+	if opts.OnlyMetadata {
+		values.Set("onlyMetadata", "true")
```

Comment:
we are still quite inconsistent when it comes to query string parameters being all lower case or camelcased

Reply:
opened an issue to fix this for the whole repo